internal/node/BUILD.bazel

```diff
@@ -43,6 +43,7 @@ filegroup(
     ]) + [
         "BUILD.bazel",
         "BUILD.nodejs_host_os_alias.tpl",
+        "//internal/node-patches:node_patches",
```

Comment:
this seems wrong - we only want to distribute the code via an npm package, not in the built-in tgz file right? To avoid users needing to update the version of the built-in. I think just remove from here?